layouts/shortcodes/newsletter/investigator-form.html

```diff
@@ -10,14 +10,18 @@
     data-ga-category="newsletters:investigator-full"
```

Comment:
Prettier only does the Tailwind class sorting stuff to a whitelisted set of files. Add `"**/shortcodes/newsletter/*.html",` to prettier.config.js and it will start sorting this file too.

Reply:
The three shortcodes are pretty similar. It might be time to extract them into a common base partial and just pass a dict of differences from each file.

Reply:
Updated
https://github.com/spotlightpa/poor-richard/pull/343/commits/b2f329f83a5d2fe01c6f8fd9fa8141f61d001ed9